Scenes/EncountersMenuScene.gd

```diff
@@ -1,16 +1,20 @@
 extends SceneBase
 
-var pickedPoolToForget = ""
+var pickedPoolToShow = ""
 var pickedFetishToChange = ""
 var pickedPersonalityStat = ""
 var pickedGenderToChange = ""
 var pickedSpeciesToChange = ""
+var npclistScene = preload("res://UI/NpcList/NpcList.tscn")
+var npclist 
```

Comment:
I don't think you need this variable to be here, it seems to be fine to declare it while creating the list inside the _run function `var npclist = npclistScene.instance()`